README.md

```diff
@@ -273,6 +273,7 @@ Then, in your `build.zig`'s `build` function, add the following before
     const zap = b.dependency("zap", .{
         .target = target,
         .optimize = optimize,
+        //.openssl = true, // required to use tls
```

Comment:
I am not entirely sure if it shouldn't read `.openssl = false;   // set to true for TLS ...`?

Reply:
Yeah, that works. I'll switch that around.